firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandlerTest.java

```diff
@@ -736,6 +738,17 @@ public void getInfo_hitsThrottleLimit_throttledStatus() throws Exception {
         .isEqualTo(firstFetchedContainer.getFetchTime());
   }
 
+  @Test
+  public void realtimeFetch_excludeEtag_andSuccess() throws Exception {
```

Comment:
I'd use `fetchWithoutEtag_` as the prefix since we're testing that method here. Can we assert the ETag header isn't sent?